documentation/common/snip-prerequisites-openshift.adoc

```diff
@@ -0,0 +1,14 @@
+:context: enmasse
```

Comment:
I think ... and we need to talk about this as a team ... you want the content to be in the file that includes the snip.  You want to define the context before you include it so that you can use it multiple times with different context each time. If the context is defined in the file, then it's always the same and can't be reused without throwing build errors because of duplicate IDs.